src/api/models/sql_models/benthicpqt.py

```diff
@@ -0,0 +1,309 @@
+from django.contrib.gis.db import models
+from django.contrib.postgres.fields import JSONField
+from django.utils.translation import ugettext_lazy as _
+
+from sqltables import SQLTableArg, SQLTableManager
+from .base import BaseSQLModel, BaseSUSQLModel, sample_event_sql_template
+
+# Unique combination of these fields defines a single (pseudo) sample unit.
+# All other fields are aggregated.
+su_fields = BaseSUSQLModel.se_fields + [
```

Comment:
Is this used, or extraneous (defined in `BenthicPhotoQuadratTransectSUSQLModel`)?